src/main/java/net/sf/mzmine/modules/peaklistmethods/identification/sirius/PeakListIdentificationTask.java

```diff
@@ -84,10 +85,18 @@
     fingerCandidates =
         parameters.getParameter(PeakListIdentificationParameters.CANDIDATES_FINGERID).getValue();
 
+    String massListName = parameters.getParameter(PeakListIdentificationParameters.MASS_LIST).getValue();
+    List<String> massLists = MassListComponent.getMassListNames();
+
     if (timer <= 0 || siriusCandidates <= 0 || fingerCandidates <= 0 || threadsAmount <= 0) {
       MZmineCore.getDesktop().displayErrorMessage(MZmineCore.getDesktop().getMainWindow(),
           "Sirius parameters can't be negative");
       cancel();
+    } else if (!massLists.contains(massListName)) {
+      MZmineCore.getDesktop().displayErrorMessage(MZmineCore.getDesktop().getMainWindow(),
+          "Mass List parameter",
+          String.format("Mass List parameter is set wrong [%s]", massListName));
+      cancel();
```

Comment:
cancel() will set the status to CANCELED, which is a different thing from ERROR. If you encounter an error, the status should be ERROR.

Reply:
Fixed. Also in SingleRowIdentificationTask